internal/jimm/access.go

```diff
@@ -804,3 +804,59 @@ func (j *JIMM) ListGroups(ctx context.Context, user *openfga.User, pagination pa
 	}
 	return groups, nil
 }
+
+// OpenFGACleanup queries OpenFGA for all existing tuples, tries to resolve each tuple and removes those
+// that JIMM cannot resolved - orphaned tuples. JIMM not being able to resolve a tuple means that the
+// corresponding entity has been removed from JIMM's database.
+func (j *JIMM) OpenFGACleanup(ctx context.Context) error {
+	for {
+		tuples, continuationToken, err := j.OpenFGAClient.ReadRelatedObjects(ctx, openfga.Tuple{}, 10, "")
+		if err != nil {
+			zapctx.Error(ctx, "reading all tuples", zap.Error(err))
+			return err
+		}
+		zapctx.Debug(ctx, "removing orphaned tuples", zap.Any("tuples", tuples))
+		orphanedTuples := j.orphanedTuples(ctx, tuples...)
+		if len(orphanedTuples) == 0 {
+			if continuationToken != "" {
+				zapctx.Error(ctx, "continue cleanup")
```

Comment:
This log line seems arbitrary and at the wrong log level.
I'd suggest to either remove it or add some info
`zapctx.Debug(ctx, "continue cleanup",zap.Int("page",page)`

Reply:
removed.